strongbox-data-service/src/main/java/org/carlspring/strongbox/data/server/InMemoryOrientDbServer.java

```diff
@@ -62,20 +62,6 @@ public void onDrop(ODatabaseInternal iDatabase)
         iDatabase.registerHook(genericEntityHook);
     }
 
-    @Override
-    public void onCreateClass(ODatabaseInternal iDatabase,
-                              OClass iClass)
-    {
-        iDatabase.registerHook(genericEntityHook);
```

Comment:
@sbespalov what do you think about this change ?

Reply:
seems that this methods was deprecated
anyway I think this shouldn't affect for us, so let it be so